neo4j/datadog_checks/neo4j/data/conf.yaml.example

```diff
@@ -1,53 +1,373 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
-  ## @param default_timeout - string - optional - default: 5
-  ## Timeout to use as a default for all your instances.
-  #
-  # default_timeout: 5
+    ## @param proxy - mapping - optional
+    ## Set HTTP or HTTPS proxies for all instances. Use the `no_proxy` list
+    ## to specify hosts that must bypass proxies.
+    ##
+    ## The SOCKS protocol is also supported like so:
+    ##
+    ##   socks5://user:pass@host:port
+    ##
+    ## Using the scheme `socks5` causes the DNS resolution to happen on the
+    ## client, rather than on the proxy server. This is in line with `curl`,
+    ## which uses the scheme to decide whether to do the DNS resolution on
+    ## the client or proxy. If you want to resolve the domains on the proxy
+    ## server, use `socks5h` as the scheme.
+    #
+    # proxy:
+    #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
+    #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    #   no_proxy:
+    #   - <HOSTNAME_1>
+    #   - <HOSTNAME_2>
+
+    ## @param skip_proxy - boolean - optional - default: false
+    ## If set to `true`, this makes the check bypass any proxy
+    ## settings enabled and attempt to reach services directly.
+    #
+    # skip_proxy: false
+
+    ## @param timeout - number - optional - default: 10
+    ## The timeout for connecting to services.
+    #
+    # timeout: 10
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
 
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param neo4j_url - string - required
     ## Neo4J url to connect to.
     #
   - neo4j_url: localhost
 
-    ## @param port - integer - optional - default: 7474
+    ## @param port - integer - required
     ## Port to connect to when reaching `neo4j_url`
     #
-    # port: 7474
+    port: 7474
 
-    ## @param user - string - required
-    ## Username to use when reaching the Neo4j instance.
+    ## @param server_name - string - required
+    ## Name of the Neo4j server to connect to.
     #
-    user: <USERNAME>
+    server_name: <SERVER_NAME>
 
-    ## @param password - string - required
-    ## Password associated with the `user` to connect to the
-    ## Neo4j instance
+    ## @param neo4j_version - string - required
+    ## The Neo4j version to gather metric from.
     #
-    password: <PASSWORD>
+    neo4j_version: <NEO4J_VERSION>
```

Comment:
Is it a config option?

Reply:
Ah nice catch- yes it doesn't appear to be used at all, I'll remove it.